auth/jwt/middleware_test.go

```diff
@@ -134,15 +134,15 @@ func TestJWTParser(t *testing.T) {
 	}
 
 	// Test for malformed token error response
-	parser = NewParser(keys, method, &jwt.StandardClaims{})(e)
+	parser = NewParser(keys, method, func() jwt.Claims { return &jwt.StandardClaims{} })(e)
```

Comment:
Similarly, this anonymous func should be provided as a top-level function in the package, e.g.

```go
// StandardClaimsFactory is a ClaimsFactory that returns an empty jwt.StandardClaims.
func StandardClaimsFactory() jwt.Claims {
    return &jwt.StandardClaims{}
}
```